apps/web/ee/lib/workflows/reminders/smsReminderManager.ts

```diff
@@ -0,0 +1,119 @@
+import {
+  WorkflowTriggerEvents,
+  TimeUnit,
+  WorkflowTemplates,
+  WorkflowActions,
+  WorkflowMethods,
+} from "@prisma/client/";
+import dayjs from "dayjs";
+import isBetween from "dayjs/plugin/isBetween";
+
+import * as twilio from "@ee/lib/workflows/reminders/smsProviders/twilioProvider";
+import smsReminderTemplate from "@ee/lib/workflows/reminders/templates/smsReminderTemplate";
+
+import prisma from "@lib/prisma";
```

Comment:
```suggestion
import prisma from "@calcom/prisma";
```

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix